### ISSUE TITLE
OpenJDK 8 broken font rendering e.g. for Google font "Great Vibes"

### ISSUE THREAD

**Issue by BuZZ-dEE** [corretto/corretto-8#176]
Please see the following images for Google font "[Great Vibes](https://fonts.google.com/specimen/Great+Vibes)":

### Screenshots
**OpenJDK 8** **_broken_** font redering:

![openjdk-8-font-incorrect](https://user-images.githubusercontent.com/125954/67096926-6529e900-f1b9-11e9-82ba-1e672216225d.png)

![openjdk-8-font-incorrect-marked](https://user-images.githubusercontent.com/125954/67101295-a58d6500-f1c1-11e9-9cc8-8dc72eb6da19.png)

**OracleJDK 8, OpenJDK 10, 11, 13** correct font rendering:

![openjdk-13-font-correct](https://user-images.githubusercontent.com/125954/67096942-6d822400-f1b9-11e9-8f39-43175ffe1189.png)

### Platform information
    OS: Debian GNU/Linux 9.9 (stretch), Ubuntu 18.04, Arch Linux
    Version: openjdk version "1.8.0_222"
             OpenJDK Runtime Environment Corretto-8.222.10.1 (build 1.8.0_222-b10)
             OpenJDK 64-Bit Server VM Corretto-8.222.10.1 (build 25.222-b10, mixed mode)

### Example code

Wrong rendering **_not_** visible:
```java
javax.swing.JButton button = new javax.swing.JButton("Open");
java.awt.Font font = new java.awt.Font("Great Vibes", Font.PLAIN, 64);
button.setFont(font);
```
![OpenJDK-8-font-size-128-wrong-rendering-not-visible png](https://user-images.githubusercontent.com/125954/67099592-8b05bc80-f1be-11e9-995b-b535ac0dda12.png)

Wrong rendering visible:
```java
javax.swing.JButton button = new javax.swing.JButton("Open");
java.awt.Font font = new java.awt.Font("Great Vibes", Font.PLAIN, 128);
button.setFont(font);
```
![OpenJDK-8-font-size-128-wrong-rendering-visible](https://user-images.githubusercontent.com/125954/67099600-8e994380-f1be-11e9-8c5a-024eb3b1809b.png)

**Comment by davecurrie** [comment]
Hi @BuZZ-dEE.

Does this happen on the latest release, 8u232 (see the "releases" section)? We're having problems reproducing the issue and it may be fixed in this version.

**Comment by BuZZ-dEE** [comment]
Hi @davecurrie you are right. Also it works correct in 8u222. As it turns out, it is related to the [font version](https://github.com/corretto/corretto-8/files/3748490/Great.Vibes.ttf.zip) we are using.

[broken font version:](https://github.com/corretto/corretto-8/files/3748490/Great.Vibes.ttf.zip)
![openjdk-8-25 222-b05_rendering_incorrect_great-vibes](https://user-images.githubusercontent.com/125954/67167520-6155bc80-f39b-11e9-8cd4-afdf1cf2757c.png)

[correct font version](https://github.com/corretto/corretto-8/files/3748496/greatvibes.zip) downloaded from [here](https://github.com/google/fonts/archive/master.zip)
![openjdk-8-1 8 0_222-b05_rendering_correct_great-vibes](https://user-images.githubusercontent.com/125954/67167522-6581da00-f39b-11e9-966f-bbe0bfa8e453.png)

**Comment by davecurrie** [comment]
Good to know you found the root cause. Closing this issue then.

**Comment by BuZZ-dEE** [comment]
@davecurrie I think this is still an issue in OpenJDK 8, because OracleJDK 8 renders the font correct. Also in Inkscape is the font rendering correct. On the other hand the font, which is not redered correct, is 
extended by some characters. So we can not just use the original Google font. Also this blocks us from switching to Coretto from OracleJDK 8.

**Comment by davecurrie** [comment]
@BuZZ-dEE Can you share the font with us?

**Comment by BuZZ-dEE** [comment]
Yes, already [uploaded](https://github.com/corretto/corretto-8/issues/176#issuecomment-544300808). 😉

https://github.com/corretto/corretto-8/files/3748490/Great.Vibes.ttf.zip

**Comment by davecurrie** [comment]
I missed that link, thanks. :)

I'm thinking it might be similar to https://github.com/corretto/corretto-8/issues/148 -- did you look to see if there are any OpenJDK bugs open for a similar issue? We will also look in any case.

**Comment by BuZZ-dEE** [comment]
We found a bug like #148 in the OpenJDK bugtracker, but it seems it is not related to this one. Also this bug seems to be in OpenJDK itself, because also AdoptOpenJDK and Alibaba Dragonwall affected. Yes, we already searched for a similar issue, but we found nothing.

**Comment by alvdavi** [comment]
Hi @BuZZ-dEE , sorry for the delay in the answer. We have confirmation this issue is fixed by:
[JDK-8057986](https://bugs.openjdk.java.net/browse/JDK-8057986)

We'll try to get this backported into OpenJDK 8u242

**Comment by ziyiluo** [comment]
This has been merged in the upstream 8u242. 8u242 release comes in Jan 2020.

**Comment by BuZZ-dEE** [comment]
Does  [openjdk8u232](
https://bugs.openjdk.java.net/browse/JDK-8235167) also get an update where this bug is fixed, or for what is the ticket [JDK-8235167
](https://bugs.openjdk.java.net/browse/JDK-8235167)?

**Comment by davecurrie** [comment]
8u232 is the current version and its update will be 8u242, where this issue is fixed. If you download the Corretto 8 update that we release on January 21st, you'll get the fix.
